src/components/MemberSearchPage/MemberTable.js

```diff
@@ -60,6 +60,8 @@ const MemberTable = props => {
   const firstItem = props.currentPage * props.membersPerPage - props.membersPerPage + 1;
   const lastItem = props.currentPage * props.membersPerPage;
 
+  const memberShowThisPage = props.showAll ? props.count.total : props.count.public;
```

Comment:
I think you have a good candidate for props destructuring, plus it will match the rest of your code style